save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/HasErrorModal.kt

```diff
@@ -160,12 +165,24 @@ val requestModalHandler: FC<PropsWithChildren> = FC { props ->
     }
 }
 
+/**
+ * [State] of request modal component
+ */
+external interface RequestModalProps : PropsWithChildren {
+    /**
+     * Currently logged in user or null
+     */
+    var userInfo: UserInfo?
+}
+
 /**
  * @property setResponse [StateSetter] for response error handler
  * @property setLoadingCounter [StateSetter] for active request counter
+ * @property setIsNeedRedirect
  */
 data class RequestStatusContext(
     val setResponse: StateSetter<Response?>,
+    val setIsNeedRedirect: StateSetter<Boolean>,
```

Comment:
Maybe call it `redirectToFallbackView` or `redirectToErrorPage`?

Reply:
Renamed